tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "include": ["src/**/*"],
+  "transpileOnly": true,
+  "skipIgnore": true,
```

Comment:
:question: I can't find these properties in the tsconfig reference, could you please share some more information on what they do/where to find reference?
https://www.typescriptlang.org/tsconfig/

Reply:
![image](https://github.com/user-attachments/assets/66cc2373-6e51-4807-bfb6-65446cb9859b)
I moved them from register.